realm/realm-library/src/main/java/io/realm/internal/TableView.java

```diff
@@ -724,17 +724,25 @@ public String toJson() {
 
     @Override
     public String toString() {
-        return nativeToString(nativePtr, 500);
-    }
+        long columnCount = getColumnCount();
+        StringBuilder stringBuilder = new StringBuilder("The TableView ");
+        stringBuilder.append("contains ");
```

Comment:
I suppose `"contains "` could be combined with `"The TableView "` in the `StringBuilder` constructor, but it's a minor thing.